src/app-layout/__integ__/app-layout-focus-delegation.test.ts

```diff
@@ -28,274 +43,316 @@ function setupTest(
     [true, false].forEach(mobile =>
       describe(`mobile=${mobile}`, () => {
-        test(
-          'should not focus panels on page load',
-          setupTest(
-            async page => {
-              await expect(page.isFocused('body')).resolves.toBe(true);
-            },
-            { pageName: 'with-split-panel', visualRefresh, mobile }
-          )
-        );
-        test(
-          'split panel focus moves to slider on open and open button on close',
-          setupTest(
-            async page => {
-              await page.click(wrapper.findSplitPanel().findOpenButton().toSelector());
-              await expect(page.isFocused(wrapper.findSplitPanel().findSlider().toSelector())).resolves.toBe(true);
-              await page.keys(['Tab', 'Tab']);
-              await expect(page.isFocused(wrapper.findSplitPanel().findCloseButton().toSelector())).resolves.toBe(true);
-              await page.keys('Enter');
-              await expect(page.isFocused(wrapper.findSplitPanel().findOpenButton().toSelector())).resolves.toBe(true);
-            },
-            { pageName: 'with-split-panel', visualRefresh, mobile }
-          )
-        );
-
-        test(
-          'tools panel focus toggles between open and close buttons',
-          setupTest(
-            async page => {
-              await page.click(wrapper.findToolsToggle().toSelector());
-              await page.keys('Enter');
-              await expect(page.isFocused(wrapper.findToolsToggle().toSelector())).resolves.toBe(true);
-              await page.keys('Enter');
-              await expect(page.isFocused(wrapper.findToolsClose().toSelector())).resolves.toBe(true);
-              await page.keys('Enter');
-              await expect(page.isFocused(wrapper.findToolsToggle().toSelector())).resolves.toBe(true);
-            },
-            { pageName: 'with-split-panel', visualRefresh, mobile }
-          )
-        );
-
-        test(
-          'navigation panel focus toggles between open and close buttons',
-          setupTest(
-            async page => {
-              // panel is closed by default on mobile
-              if (mobile) {
-                await page.click(wrapper.findNavigationToggle().toSelector());
-              }
-              await page.click(wrapper.findNavigationClose().toSelector());
-              await page.keys('Enter');
-              await expect(page.isFocused(wrapper.findNavigationClose().toSelector())).resolves.toBe(true);
-              await page.keys('Enter');
-              await expect(page.isFocused(wrapper.findNavigationToggle().toSelector())).resolves.toBe(true);
-              await page.keys('Enter');
-              await expect(page.isFocused(wrapper.findNavigationClose().toSelector())).resolves.toBe(true);
-            },
-            { pageName: 'with-split-panel', visualRefresh, mobile }
-          )
-        );
-
-        test(
-          'drawers focus toggles between open and close buttons',
```

Comment:
What happened to this test?

Reply:
it was split into 2 separate tests, one for opening a wide drawer, another for a narrow drawer. It has since been combined and added back almost as the original